common/functions/src/scalars/uuids/uuid_creator.rs

```diff
@@ -80,22 +79,28 @@ impl UUIDCreator for UUIDZero {
     }
 }
 
-impl<T> Function for UUIDCreatorFunction<T>
+impl<T> Function2 for UUIDCreatorFunction<T>
 where T: UUIDCreator + Clone + Sync + Send + 'static
 {
     fn name(&self) -> &str {
         self.display_name.as_str()
     }
 
-    fn return_type(&self, _args: &[DataTypeAndNullable]) -> Result<DataTypeAndNullable> {
-        let dt = DataType::String;
-        Ok(DataTypeAndNullable::create(&dt, false))
+    fn return_type(
+        &self,
+        _args: &[&common_datavalues2::DataTypePtr],
+    ) -> Result<common_datavalues2::DataTypePtr> {
+        Ok(StringType::arc())
     }
 
-    fn eval(&self, _columns: &DataColumnsWithField, input_rows: usize) -> Result<DataColumn> {
+    fn eval(
+        &self,
+        _columns: &common_datavalues2::ColumnsWithField,
+        _input_rows: usize,
+    ) -> Result<common_datavalues2::ColumnRef> {
         let uuid = T::create();
-        let value = DataValue::String(Some(uuid.to_string().as_bytes().to_vec()));
+        let col = StringColumn::new_from_slice(vec![uuid.to_string()]);
```

Comment:
shoule be  `ConstColumn::new( single_column, input_rows )`

Reply:
Why do we need `ConstColumn`?

Reply:
Because we need to know the size of the column. If we use `StringColumn::new_from_slice(vec![uuid.to_string()])` , the size is 1. Columns inside the same block must have the same size.